docs/user_guide/use_case/custom_dataloaders.md

```diff
@@ -130,7 +132,14 @@ model.train(
 )
 ...
 ```
-Key Differences between them in terms of Custom Dataloaders:
+
+3. In addition to those we also added a {class}`scvi.dataloaders.CollectionAdapter` dataloader which is a scvi-tools wrapper for Anndata's AnnCollection.
```

Comment:
In addition to those we also added a {class}`scvi.dataloaders.CollectionAdapter` dataloader which is an scvi-tools wrapper for AnnData's AnnCollection.
This API is intended for training models on a set of AnnData files. AnnCollection creates a layy concatenation of disk-backed files. The dataloader jointly subsets these datasets along the observations axis.
This wrapper mimics the standard API, soo in practice, users wrap their collection objects then proceed with the scvi-tools workflow as normal and not using the custom dataloader workflow.
Note that in order to use it count data should be in sparse form (sparse.csr_matrix).